kibana/Dockerfile

```diff
@@ -22,5 +22,5 @@ LABEL io.astronomer.docker.build.number=$BUILD_NUMBER
 
 # to fix bind-license security issues
 USER root
-RUN yum -y update bind-license
+RUN yum -y update bind-license dbus
```

Comment:
Same as above comment. Not a NATS change.

Reply:
done https://github.com/astronomer/ap-vendor/pull/40